src/llvm/LLVMDG2Dot.h

```diff
@@ -6,6 +6,8 @@
 #include "DG2Dot.h"
 #include "llvm/LLVMNode.h"
 
+#include "llvm/DebugInfo.h"     //DIScope
+
```

Comment:
In LLVM 4.0 (and probably already in some earlier versions, haven't checked that), the file was moved to `llvm/IR/DebugInfo.h`